lib/js/components/Form/FormField/FormField.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<div class="ds-formField">
+		<div v-if="label || subLabel || $slots.labelAside" class="ds-formField__labels">
+			<div class="ds-formField__labelRow">
+				<div class="ds-formField__labelWrapper">
+					<label v-if="label" class="ds-formField__label" :for="id">
+						<span class="ds-formField__labelText">{{ label }}</span>
+						<span
+							v-if="isRequired"
+							class="ds-formField__labelRequired"
+							aria-hidden="true"
+						>
+							*
+						</span>
+						<span v-if="labelInfo" class="ds-formField__labelInfo">{{
+							labelInfo
+						}}</span>
+					</label>
+					<slot name="help"></slot>
+				</div>
+				<div v-if="$slots.labelAside" class="ds-formField__labelAside">
+					<slot name="labelAside"></slot>
+				</div>
+			</div>
+			<div class="ds-formField__subLabelRow">{{ subLabel }}</div>
+		</div>
+		<div class="ds-formField__mainRow">
+			<slot name="field" :field-id="id" :message-id="messageId"></slot>
+		</div>
+		<div v-if="$slots.message || $slots.fieldStatus" class="ds-formField__footerRow">
+			<div v-if="$slots.message" class="ds-formField__message">
```

Comment:
when `message` slot is not provided, then `formField__fieldStatus` is rendered on the lest instead of right side